src/doc/trpl/structs.md

```diff
@@ -87,3 +87,32 @@ fn main() {
     point.y = 6; // this causes an error
 }
 ```
+
+# Update syntax
+
+A `struct` can include `..` to indicate that you want to update just some
+values of a struct. For example:
```

Comment:
... do people use this syntax a lot for the case of updating the same struct?

I had always assumed that one would use assignment statements for that rather than FRU, and that FRU was more generally used for constructing new struct values.  (Just thought it was funny that the documentation takes the opposite perspective on what this is used for.)